src/sinks/splunk_hec/logs/integration_tests.rs

```diff
@@ -525,10 +525,7 @@ async fn splunk_auto_extracted_timestamp() {
         // Splunk will determine the timestamp from the *message* field in the event.
         // Thus, we expect the `timestamp` field to still be present.
         assert_eq!(
```

Comment:
🗒️ `cargo vdev -v int start -a splunk` works in Ubuntu. Also, when working on Ubuntu `cargo vdev -v int test --retries 2 -a splunk` fails to runs tests.